tests/pulse/test_circuit_to_pulses.py

```diff
@@ -462,17 +503,14 @@ def test_translate(self, platform):  # pylint: disable=R0914 # disable pyling to
         assert isinstance(pulse_schedules[0], PulseSchedule)
 
         pulse_schedule = pulse_schedules[0]
-        # there are 6 different buses + 3 empty for unused flux lines
-        assert len(pulse_schedule) == 9
-        assert all(len(schedule_element.timeline) == 0 for schedule_element in pulse_schedule.elements[-3:])
-
-        # we can ignore empty elements from here on
-        pulse_schedule.elements = pulse_schedule.elements[:-3]
+        # there are 7 pulse schedules
+        assert len(pulse_schedule) == 7
 
         # extract pulse events per bus and separate measurement pulses
         pulse_bus_schedule = {
             pulse_bus_schedule.port: pulse_bus_schedule.timeline for pulse_bus_schedule in pulse_schedule
         }
+
```

Comment:
This shouldn't change, right? 🤔

Reply:
That is what I don't fully understand from the mocked runcard, there are two schedules less being created. If any of the flux lines do not have an AWG there should not be empty schedules added, right?

Reply:
Amazing, thanks!!